sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageAggregator.java

```diff
@@ -65,11 +65,11 @@ public CoverageAggregator(CoverageConfiguration coverageConf, Configuration conf
 
   @VisibleForTesting
   CoverageAggregator(CoverageConfiguration coverageConf, Configuration configuration,
-                     CoverageCache coverageCache,
```

Comment:
This does not respect our convention, I remember that we have agreed to have "each parameter on single line, aligned with the 1st one"

Reply:
Same as https://github.com/SonarSource/sonar-dotnet/pull/3531#discussion_r473833795